hooks/useElementPositionAndSize.ts

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from 'react';
```

Comment:
```suggestion
import React, { useState, useEffect } from 'react';
```

SInce you are using `React` in following lines

Reply:
Are we importing react on all other components? If yes, we should enforce the React import then :)

Reply:
This import is for `React.RefObject` usage.

Reply:
Yeah I saw that, when I rebased your PR I saw that.